test/gitlab/configuration.test.ts

```diff
@@ -92,3 +92,47 @@ test("throws when adding an existing includes", () => {
     /already contains one or more templates specified in/
   );
 });
+
+test("respected the original format when variables are added to jobs", () => {
+  // GIVEN
+  const p = new TestProject({
+    stale: true,
+  });
+  new CiConfiguration(p, "foo", {
+    jobs: {
+      build: {
+        variables: { AWS_REGION: "eu-central-1" },
+      },
+    },
+  });
+  const snapshot = synthSnapshot(p);
+  // THEN
+  expect(snapshot[".gitlab/ci-templates/foo.yml"]).toContain(
+    `# ~~ Generated by projen. To modify, edit .projenrc.js and run "npx projen".
+
+build:
+  variables:
+    AWS_REGION: eu-central-1
+`
+  );
```

Comment:
Super small nit - let's just assert that it contains the caps variable and ignore the rest, so that this test doesn't get broken by formatting changes and such.
```suggestion
  expect(snapshot[".gitlab/ci-templates/foo.yml"]).toContain("AWS_REGION: eu-central-1");
```

Reply:
done